R/computeAnnualDegreeDays.R

```diff
@@ -83,6 +83,9 @@ initCalculation <- function(fileMapping,
   hddcdd <- do.call("rbind", lapply(seq(1, yEnd - yStart + 1), function(i) {
     message("Initiating calculating degree days for the year: ", seq(yStart, yEnd)[[i]])
 
+    # add year tag
+    fileNames <- lapply(fileNames, function(x) gsub("\\.nc$", paste0("_", i, ".nc"), x))
```

Comment:
Absolutely irrelevant here. Just FYI, you should use `sub` instead of `gsub` if you don't expect multiple pattern replacements as it is faster.
```suggestion
    fileNames <- lapply(fileNames, function(x) sub("\\.nc$", paste0("_", i, ".nc"), x))
```